presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/PackageApiUtils.java

```diff
@@ -69,4 +72,35 @@ public static boolean isRunning()
     {
         return isInstalled() && executeCommand("service", "presto", "status") == 0;
     }
+
+    public static void untarFile(File configTar, Path destinationDir)
+            throws PrestoManagerException
+    {
+        int untarResult = executeCommand("tar", "-xzvf", configTar.getAbsolutePath(), "-C", destinationDir.toAbsolutePath().toString());
+        if (untarResult != 0) {
+            throw new PrestoManagerException(format("Failed to untar the file: %s", configTar), untarResult);
+        }
+    }
+
+    public static void deleteDirContents(Path directory)
+            throws PrestoManagerException
+    {
+        try {
+            deleteDirectoryContents(directory);
+        }
+        catch (IOException e) {
```

Comment:
Could be fine-grained. See [Doc](https://google.github.io/guava/releases/21.0/api/docs/com/google/common/io/MoreFiles.html#deleteDirectoryContents-java.nio.file.Path-com.google.common.io.RecursiveDeleteOption...-)